src/sage/rings/rational_field.py

```diff
@@ -1663,7 +1664,7 @@ def _factor_univariate_polynomial(self, f):
         G = list(f._pari_with_name().factor())
 
         # normalize the leading coefficients
-        F = [(f.parent()(g).monic(), int(e)) for (g,e) in zip(*G)]
+        F = [(f.parent()(g).monic(), int(e)) for (g, e) in zip(*G)]
```

Comment:
```suggestion
        P = f.parent()
        F = [(P(g).monic(), int(e)) for (g, e) in zip(*G)]
```
Can we do one micro-optimization here since we are already modifying this line?